package.json

```diff
@@ -17,7 +17,7 @@
     "lint": "eslint **/*.js",
     "lint:fix": "eslint **/*.js --fix",
     "lint-staged": "lint-staged",
-    "test": "nyc --reporter=lcov mocha tests/**/*.js",
+    "test": "nyc --reporter=lcov mocha tests/**/no-japanese-character.js",
```

Comment:
Looks like we missed a spot

Reply:
Thanks for noticing!
I finished editing it.
https://github.com/chejen/eslint-plugin-i18n/pull/20/commits/0adee308ad79780d29c9b2a90ae416bbc81fd6d0